Intersect.Client.Core/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -475,6 +476,8 @@ public override void DrawString(
         Color? borderColor = null
     )
     {
+        // Apply Arabic text fixing if needed
+        text = ArabicFixer.FixIfNeeded(text) ?? text;
```

Comment:
This is an elegant solution in terms of line count but dreadful for performance.

At the absolute least this needs to be an opt-in setting in the client config.json.

Ideally strings would be fixed one time, as they are set instead of every single frame when we go to render them.

Also I don't have any knowledge of Arabic, but if these fixes can shorten or lengthen the string then you will want this in the MeasureString functions as well otherwise you will have UI elements not sizing appropriately, cut off text in chat bubbles, etc.